index.html

```diff
@@ -1,259 +1,241 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <meta name="description" content="">
-    <meta name="author" content="">
-
-    <title>THE SQUEEZYLEMON</title>
-
-    <!-- Bootstrap Core CSS -->
-    <link href="css/bootstrap.min.css" rel="stylesheet">
-
-    <!-- Custom CSS -->
-    <link href="css/grayscale.css" rel="stylesheet">
-
-    <!-- Custom Fonts -->
-    <link href="font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
-    <link href="http://fonts.googleapis.com/css?family=Lora:400,700,400italic,700italic" rel="stylesheet" type="text/css">
-    <link href="http://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet" type="text/css">
-
-    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
-    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
-    <!--[if lt IE 9]>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta name="description" content="">
+        <meta name="author" content="">
+
+        <title>THE SQUEEZYLEMON</title>
```

Comment:
Hey Lady,

How are you? We arrived yesterday in the evening, slept on the floor at
some friends and today finally in a nice guesthouse. Next time we take the
plane, what a horrible bus ride (no aircon). Thanks for you messages, we're
going to change it all!

Did you send out the app? Already got some responses?

Cheers!

Nils & Bouke


2015-05-05 16:36 GMT+07:00 Alexej <notifications@github.com>:

> In index.html
> <https://github.com/nilsvanbreen/thesqueezylemon.com/pull/1#discussion_r29654619>
> :
>
> > -    <!-- Custom Fonts -->
> > -    <link href="font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
> > -    <link href="http://fonts.googleapis.com/css?family=Lora:400,700,400italic,700italic" rel="stylesheet" type="text/css">
> > -    <link href="http://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet" type="text/css">
> > -
> > -    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
> > -    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
> > -    <!--[if lt IE 9]>
> > +    <head>
> > +        <meta charset="utf-8">
> > +        <meta http-equiv="X-UA-Compatible" content="IE=edge">
> > +        <meta name="viewport" content="width=device-width, initial-scale=1">
> > +        <meta name="description" content="">
> > +        <meta name="author" content="">
> > +
> > +        <title>THE SQUEEZYLEMON</title>
>
> Add a compelling title. Something like "THE SQUEEZYLEMON - Creative Agency
> Bla Bla". Use this tool <http://snippetoptimizer.net/> to help you with
> character limits. Also fill out the description.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nilsvanbreen/thesqueezylemon.com/pull/1/files#r29654619>
> .
>